packages/navie/src/commands/review-command.ts

```diff
@@ -0,0 +1,138 @@
+import Command, { CommandRequest } from '../command';
+import Message from '../message';
+import CompletionService from '../services/completion-service';
+import LookupContextService from '../services/lookup-context-service';
+import { UserContext } from '../user-context';
+import { ExplainOptions } from './explain-command';
+import { ContextV2 } from '../context';
+
+const BASE_PROMPT_GENERIC = `You are an expert software engineer conducting a code review on a git diff provided for a pull request. Your task is to analyze the changes thoroughly and provide constructive, professional feedback. When reviewing the diff, consider the following aspects:
+
+- **Correctness**: Identify any bugs, logical errors, or edge cases that may cause the code to fail.
+- **Code Quality**: Assess readability, maintainability, and adherence to coding standards and style guidelines.
+- **Performance**: Point out any inefficiencies or opportunities for optimization.
+- **Security**: Highlight potential security vulnerabilities or risks introduced by the changes.
+- **Documentation**: Ensure that the code is well-documented where necessary, including comments and documentation updates.
+- **Testing**: Check if the changes are adequately covered by unit tests and suggest additional tests if needed.
+- **Compatibility**: Consider the impact on existing functionality, including backward compatibility and integration with other system components.
+- **Design and Architecture**: Evaluate whether the code follows good design principles and fits well within the overall architecture.`;
```

Comment:
Is there a way to make the output compact when Navie decides that a particular section is OK as is? Reading `r/codereview` (example: https://www.reddit.com/r/codereview/comments/1gaj3vg/what_ai_functionality_would_you_want_in_a_code/) a primary complaint is reviews that are too verbose. So it would be best if Navie can avoid giving a verbose answer when really there are no major concerns. 

Is Navie able to emit `N/A`, `OK`, or `No areas of substantial concern` for a particular section?